src/xml.tsx

```diff
@@ -78,7 +78,10 @@ export function SvgXml(props: XmlProps) {
   }
 }
 
-export async function fetchText(uri: string) {
+export async function fetchText(uri?: string) {
+  if (!uri) {
+    return null;
+  }
```

Comment:
I believe we should check for that earlier

Reply:
I believe it's better to check the URI only in that specific place, especially if we have multiple areas where we utilize that function.